### PR TITLE
forward port changes from 6.1

### PR DESCRIPTION
**Pull request by lmignon**
A forward port from 6.1. I'm working on a 7.0 branch  https://github.com/acsone/geospatial/tree/7.0-migrate-from-6.1. At this stage, the widget works fine also in 7.0.

### CODE REVIEW

File: base_geoengine/__openerp__.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 #
 ##############################################################################
 {'name': 'Geospatial support for OpenERP',
- 'version': '0.1',
+ 'version': '0.2.0',
```
Comment (pedrobaeza) [inline base_geoengine/__openerp__.py]: Put only two numbers for version.

**Comment by pedrobaeza** [comment]
:+1:

**Comment by yvaucher** [comment]
:+1:

**Comment by lepistone** [comment]
ok! :+1:

**Pull request merged**